zappa/handler.py

```diff
@@ -336,9 +336,27 @@ def handler(self, event, context):
 
         # This is a direct command invocation.
         elif event.get('command', None):
+            if not event['command']:
```

Comment:
`event.get('command')` is the same as `event['command']`, so it is a check for falsity, with a side effect of returning `None` instead of throwing `KeyError`